pandas/tests/groupby/test_grouping.py

```diff
@@ -501,15 +501,18 @@ def test_groupby_level(self, sort, mframe, df):
         with pytest.raises(ValueError, match=msg):
             df.groupby(level=1)
 
-    def test_groupby_level_index_names(self):
+    @pytest.mark.parametrize("axis", [0, 1])
```

Comment:
Parametrize over `'index', and `'columns'` too. We may have a fixture for that, not sure.

Reply:
axis it is

Reply:
You mean parametrize with [0, 1, 'index', 'columns'] ?

Reply:
no, simply include 'axis' in your keyword list and it will just work as its a fixture

Reply:
done, thanks